libs/designer-ui/src/lib/searchabledropdownWithAddAll/index.tsx

```diff
@@ -41,11 +41,17 @@ export const SearchableDropdownWithAddAll: FC<SearchableDropdownWithAddAllProps>
     ) : null;
   };
 
+  const labelId = `dropdown-label-${label ? label.replace(' ', '-').toLowerCase() : 'no-label'}`;
```

Comment:
if there isn't a label, can we put labelId = undefined since that way SearchDropdown won't have a labelId as well

Reply:
done!